gui/src/components/Taipy/TreeView.tsx

```diff
@@ -101,8 +121,8 @@ const renderTree = (lov: LovItem[], active: boolean, searchValue: string, select
                 nodeId={li.id}
                 label={typeof li.item === "string" ? li.item : "undefined item"}
                 disabled={!active}
-                allowSelection={selectLeafsOnly ? (!children || children.length == 0) : true}
-                lovIcon={typeof li.item !== "string" ? li.item as Icon : undefined}
+                allowSelection={selectLeafsOnly ? !children || children.length == 0 : true}
```

Comment:
I did prefer the parenthesis that make the expression easier to read.
But if linters complain... who am I?